app/code/FrontCommerce/ChronorelaisHeadless/Model/RelayPoint.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace FrontCommerce\ChronorelaisHeadless\Model;
+
+use Chronopost\Chronorelais\Helper\Webservice as HelperWebservice;
+use Exception;
+use Magento\Quote\Model\Quote\AddressFactory;
+
+class RelayPoint
+{
+    protected AddressFactory $addressFactory;
+    protected HelperWebservice $helperWebservice;
+
+    public function __construct(
+        AddressFactory   $addressFactory,
+        HelperWebservice $helperWebservice
+    )
+    {
+        $this->addressFactory = $addressFactory;
+        $this->helperWebservice = $helperWebservice;
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function getRelayPoints(
+        string $methodCode,
+        string $postcode,
+        string $countryId,
+        string $city,
+        array  $street
+    ): array
+    {
+        $street = $street ? implode(' ', $street) : '';
```

Comment:
unless I'm mistaken, it can be simplified with:

```suggestion
        $street = implode(' ', $street);
```